modules/standard/IO.chpl

```diff
@@ -1965,6 +1965,30 @@ proc =(ref lhs:channel, rhs:channel) {
   lhs._channel_internal = rhs._channel_internal;
 }
 
+pragma "no doc"
+proc ==(const ref a: channel, const ref b: channel): bool {
+  if a.writing != b.writing  then return false;
+  if a.kind != b.kind then return false;
+  if a.locking != b.locking then return false;
+  if a.home != b.home then return false;
+  if a._channel_internal != b._channel_internal then return false;
+  if a._readWriteThisFromLocale != b._readWriteThisFromLocale then return false;
+
+  return true;
+}
+
+pragma "no doc"
+proc !=(const ref a: channel, const ref b: channel): bool {
+  if a.writing == b.writing then return false;
```

Comment:
Seems to me that this should just return `! (a == b)` to run the above function.

Reply:
@mppf I have done the same. Kindly have a look at it. Thanks

Reply:
Looks good, thanks